src/query/functions/aggregation/take_test.go

```diff
@@ -111,9 +172,9 @@ func TestTakeTopFunctionFilteringWithoutA(t *testing.T) {
 	}
 
 	// Should have the same metas as when started
-	assert.Equal(t, seriesMetas, sink.Metas)
+	require.Equal(t, seriesMetas, sink.Metas)
```

Comment:
Any reason for switching these from assert to require?

Reply:
These tests were already using `require` so my intention was to unify assertions. I can for sure replace some checks to use `assert` so that failing tests wont stop further test execution.